### ISSUE TITLE
ngd as a module: quiet mode

### ISSUE THREAD

**Issue by asetGem** [kblin/ncbi-genome-download#96]
Hi, 

I am using ncbi-genome-download to download genomes from a python script. 
When I use it in a simple script, it works: it downloads the genomes, and then stops, without anything in stdout. 
However, when I use it in a script where a logger has been set to the info level:

```
logger = logging.getLogger('')
logger.setLevel(logging.INFO)
```
It also works, but writes, for each genome, in stdout: 

```
Checking record 'GCF_000964205.1'
Checking record 'GCF_002358055.1'
```
(with `setLevel(logging.DEBUG)` I have even more details (comparison with each taxid), but this is ok as it is for debug mode.)

All 'checking record*' line is considered as INFO.

The problem is that I am downloading a lot of genomes, and I cannot have 1 'checking' line for each of them, as the stdout won't be readable...

Do you have an idea on how to put it quiet?

Thanks,

A.

**Comment by kblin** [comment]
Hi there,

I don't think this is straightforward with the current code, as that is logging to the root logger. I'm happy to fix that, though.

**Comment by kblin** [comment]
Ok, with 3e81eb4 this should be fixed.

If you use something like in the attached example, you should be able to selectively set the ncbi-genome-download log level.

[example.py.txt](https://github.com/kblin/ncbi-genome-download/files/3565312/example.py.txt)

Basically, you want the
`logging.getLogger("ncbi-genome-download").setLevel(logging.WARNING)`
line and use whatever level you want.

**Comment by kblin** [comment]
Whoops, I wanted to leave this open until the fix is in a release.

**Comment by asetGem** [comment]
Thanks for your answer! 

I was about to post an update, as, in the meanwhile, I found how to deal with that.
I named my logger: `logger = logging.getLogger('myapp') `(to use something else than the root logger which is already used in ngd)
Then, I used `logger.propagate = False`, so that the ngd root logger does not influence my own logger.

But yes, I think the best way would be to rename the logger used by ngd, so that it does not log anything to the root logger.
